projects.md

```diff
@@ -0,0 +1,67 @@
+# Projects
+
+## Sales Prospect & Internal Proposals
+
+While the Business Planning WG is mainly in charge of sales lead generation,
+all members may pursue leads as they come up.
+These may be a client or a grant opportunity.
+Members are strongly encouraged to record leads into the co-operative's [leads tracking system](https://link.hypha.coop/leads) whether or not we want to pursue this lead.
+
+If a member decides to pursue a **lead**,
+they can reach out for preliminary discussions to see whether there is a potential to work together.
```

Comment:
```suggestion
they can reach out to them to see whether there's potential to work together.
```

I was at first reading this as reaching out internally to see if anyone wants to collab.